### ISSUE TITLE
Remove image as well as comment

### ISSUE THREAD

**Issue by malteserteresa** [opt-out-tools/opt-out#77]
**Objective**
Remove the image associated with misogynistic comment

**Description**
Currently we don't have the capacity to analyse images too. So if a comment is classified as misogynistic, we will automatically remove the image too.

**Skills**

**Tools**

**Time Estimation**

**Tasks**
Not sure on how to do this.

**Comment by river-honer** [comment]
Hi there :) Do you want help to achieve this, or would you just like it done? This seems like something I could help with, and if you have time this weekend, we could go over it.

**Comment by malteserteresa** [comment]
River I would love to go over it, any chance to learn more! Would you have
time Sunday?

**Comment by river-honer** [comment]
Yes, I'm free this sunday, we can do it in person or remote :) Any time is fine with me. I think we can figure this out!

**Comment by ribicn-softerrific** [comment]
After the latest testing we have 3 cases:

1. Crossed thought: the text is crossed, **images are shown**
2. Make text invisible: the color of a text is the same as the background, **images are shown**
3. Completly removed: the text is removed, **images are removed**

Do we leave it as is, or do images need to be removed on options 1,2??

**Comment by ribicn-softerrific** [comment]
Just remove it for all 3 cases.

**Comment by river-honer** [comment]
This raises an interesting issue about how the display options input component only describe the display options of the text but not the image. I will make an issue to think about this

**Comment by river-honer** [comment]
Move to next milestone.

**Comment by ribicn-softerrific** [comment]
It is set to remove the emoji image as well as an image from tweets that are misogynistic and marked by text-white and text-crossed. I recommend setting images as crossed in text-crossed mode, as it will be more intuitive to the users.
